### PR TITLE
LEAN-2407

### PR DESCRIPTION
**Pull request by asouqi**

### CODE REVIEW

**Comment by mnatsheh** [comment]
There are some failing tests.

**Comment by asouqi** [comment]
> There are some failing tests.

fix

File: package.json [hunk 1]
```diff
@@ -30,6 +30,7 @@
   },
   "dependencies": {
     "@manuscripts/json-schema": "^2.0.0",
+    "@manuscripts/data": "1.0.1",
```
Comment (mnatsheh) [inline package.json:33]: I believe that there is a 1.0.2 version of data, so let's use that. Also, you should have a caret there.

**Review by mnatsheh — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (mnatsheh) [reply package.json:33]: Actually, we already have https://github.com/Atypon-OpenSource/manuscripts-transform/pull/18 which updates manuscripts-data to 1.0.2. We should just wait for that.

**Pull request closed without merging**